config/jobs/kubernetes-sigs/sig-windows/windows-gce.yaml

```diff
@@ -0,0 +1,144 @@
+presets:
+- labels:
+    preset-load-gce-windows: "true"
+  env:
+  - name: MASTER_SIZE
+    value: "n1-standard-4"
+  - name: NUM_WINDOWS_NODES
+    value: "1"
+  - name: NUM_NODES
+    value: "1"
+  - name: NODE_SIZE
+    value: "n1-standard-8"
+  - name: NODE_DISK_SIZE
+    value: "100GB"
+  # Ensure good enough architecture for master machines.
+  - name: MASTER_MIN_CPU_ARCHITECTURE
+    value: "Intel Broadwell"
+  - name: MAX_PODS_PER_NODE
+    value: "128"
+  - name: PREPULL_TIMEOUT
+    value: "10m"
+  # APIResponsiveness test depends on Prometheus server which couldn't scrape kube-proxy,
+  # unless it’s binding 0.0.0.0 as per https://github.com/helm/charts/issues/16476
+  # Reference: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml#L127
+  - name: KUBEPROXY_TEST_ARGS
+    value: "--profiling --metrics-bind-address=0.0.0.0"
+  - name: WINDOWS_NODE_OS_DISTRIBUTION
+    value: "win2019"
```

Comment:
Windows 2019 is end of life, is it possible to update this to 2022?
We don't test 2019 against recent K8s branches anymore either...

Reply:
nevermind, i see you have a 2022 job too.